src/components/Modals/Send/Form.tsx

```diff
@@ -19,7 +19,8 @@ import { QrCodeScanner } from './views/QrCodeScanner'
 
 export type SendInput<T extends ChainId = ChainId> = {
   [SendFormFields.AccountId]: AccountId
-  [SendFormFields.Address]: string
+  [SendFormFields.To]: string
+  [SendFormFields.From]: string
```

Comment:
don't change this at all, but i just want to point out this concept of using hidden form fields as state management was a mistake and we should never do this in new code.